### PR TITLE
Bababooey but less often now

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

Alternative to #51570 because I don't hate fun. Adds a 60 second cooldown to these emotes.

## Why It's Good For The Game

bababooey

## Changelog
:cl:
tweak: You can only invoke bababooey once a minute now.
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
remove the useless parenthesis around them

**Comment by Qustinnus** [comment]
ugh wrong button why do i phonepost at 6am

**Comment by wesoda25** [comment]
bababooey should be a 10 point positive quirk

**Comment by ATH1909** [comment]
alternative proposition that I think someone else suggested: you can only bababooey once per round. use your bababooey wisely.

**Comment by ATH1909** [comment]
but yeah, this works too

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -604,6 +609,7 @@
 	message = "spews something softly."
 	emote_type = EMOTE_AUDIBLE
 	vary = TRUE
+	cooldown = (60 SECONDS)
 	mob_type_allowed_typecache = list(/mob/living/carbon/human, /mob/living/brain, /mob/living/silicon)
 
 /datum/emote/living/ffff/get_sound(mob/living/user)
```
Comment (ATH1909) [inline code/modules/mob/living/emote.dm:615]: add a cooldown to ffff as well

Comment (ATH1909) [reply code/modules/mob/living/emote.dm:615]: unless you, too, are a man of culture who likes the gloves of the north star *ffff battlecry meme

**Comment by tralezab** [comment]
Vekter I don't hate fun, I just hate when people have fun. Pain is never the fault of the feeler, just the fault of all the actions you consciously chose to take.

File: code/modules/mob/living/emote.dm (hunk 1 above)
Comment (TheVekter) [reply code/modules/mob/living/emote.dm:615]: That's... that's what that is, yes.

Comment (ATH1909) [reply code/modules/mob/living/emote.dm:615]: wait, I didn't realize that you did, in fact, add a cooldown

Comment (ATH1909) [reply code/modules/mob/living/emote.dm:615]: never mind, then

**Comment by Fikou** [comment]
60 seconds is way too long, it means i can bababooey only about 50 times in any given round

**Comment by ATH1909** [comment]
> 60 seconds is way too long, it means i can bababooey only about 50 times in any given round

Maybe make it 10 seconds instead?

**Comment by Ziiro** [comment]
Make it cause brain damage

**Comment by bluetshirtguy** [comment]
shitty memes shouldn't be added into ss13 anymore, its just spam until people are sick of the joke. let it die.

**Comment by Kelenius** [comment]
> shitty memes shouldn't be added into ss13 anymore, its just spam until people are sick of the joke. let it die.

**Comment by Doosteroni** [comment]
> shitty memes shouldn't be added into ss13 anymore, its just spam until people are sick of the joke. let it die.

You know this is...tg right?

**Comment by optimumtact** [comment]
I am going to remove it

**Pull request closed without merging**